docs/machine-learning/tutorials/image-classification.md

```diff
@@ -111,336 +111,249 @@ toaster2.png	appliance
+    [!code-csharp[ReadFromTsv](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#ReadFromTsv)]
 
-The `ReuseAndTuneInceptionModel()` method executes the following tasks:
+    The code parses through the `tags.tsv` file to add the file path to the image file name for the `ImagePath` property and load it and the `Label` into an `ImageData` object.
 
-* Loads the data
-* Extracts and transforms the data.
-* Scores the TensorFlow model.
-* Tunes (retrains) the model.
-* Displays model results.
-* Evaluates the model.
-* Returns the model.
+### Create a method to make a prediction
 
-Create the `ReuseAndTuneInceptionModel()` method, just after the `InceptionSettings` struct and just before the `DisplayResults()` method, using the following code:
+1. Create the `ClassifySingleImage()` method, just before the `DisplayResults()` method, using the following code:
 
-```csharp
-public static ITransformer ReuseAndTuneInceptionModel(MLContext mlContext, string dataLocation, string imagesFolder, string inputModelLocation, string outputModelLocation)
-{
-
-}
-```
+    ```csharp
+    public static void ClassifySingleImage(MLContext mlContext, ITransformer model)
+    {
 
-### Load the data
+    }
+    ```
 
-Data in ML.NET is represented as an [IDataView class](xref:Microsoft.ML.IDataView). `IDataView` is a flexible, efficient way of describing tabular data (numeric and text). Data can be loaded from a text file or in real time (for example, SQL database or log files) to an `IDataView` object.
+1. Create an `ImageData` object that contains the fully qualified path and image file name for the single `ImagePath`. Add the following code as the next  lines in the `ClassifySingleImage()` method:
 
-Load the data using the `MLContext.Data.LoadFromTextFile` wrapper. Add the following code as the next line in the `ReuseAndTuneInceptionModel()` method:
+    [!code-csharp[LoadImageData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#LoadImageData)]
 
-[!code-csharp[LoadData](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#LoadData "Load the data")]
+1. Make a single prediction, by adding the following code as the next line in the `ClassifySingleImage` method:
 
-### Extract Features and transform the data
+    [!code-csharp[PredictSingle](../../../samples/machine-learning/tutorials/TransferLearningTF/Program.cs#PredictSingle)]
 
-Pre-processing and cleaning data are important tasks that occur before a dataset is used effectively for machine learning.  Using data without these modeling tasks can produce misleading results.
+    The [PredictionEngine class](xref:Microsoft.ML.PredictionEngine%602) is a convenience API that performs a prediction on a single instance of data. The [Predict()](xref:Microsoft.ML.PredictionEngine%602.Predict%2A) function makes a prediction on a single column of data.
```

Comment:
Suggestion:

The [PredictionEngine class](xref:Microsoft.ML.PredictionEngine%602) is a convenience API that performs a prediction on a single instance of data. To get the prediction, use the [Predict()](xref:Microsoft.ML.PredictionEngine%602.Predict%2A) method.

Reply:
Changed